WatchFaceAlphaKotlin/app/src/main/java/com/example/android/wearable/alpha/AnalogWatchCanvasRenderer.kt

```diff
@@ -115,8 +115,14 @@ class AnalogWatchCanvasRenderer(
     private var currentWatchFaceSize = Rect(0, 0, 0, 0)
 
     init {
+
+        Log.d(TAG, "init { }")
+
         scope.launch {
             analogWatchFaceViewModel.uiState.collect { uiState ->
+
+                Log.d(TAG, "collect worked! : $uiState")
```

Comment:
Might be useful to remove some of the logging once everything is working.

Reply:
I will do a pass after the things are addressed and strip out the Log.d()